src/native/managed/cdacreader/Microsoft.Diagnostics.DataContractReader.Contracts/Contracts/EcmaMetadata.cs

```diff
@@ -0,0 +1,19 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Reflection.Metadata;
+
+namespace Microsoft.Diagnostics.DataContractReader.Contracts;
+
+internal sealed class FEcmaMetadata : IContractFactory<IEcmaMetadata>
```

Comment:
I'm not familiar with the "F" prefix. Windows conventions historically used "C", but they are uncommon in .NET and less than helpful. Do we need a prefix? Why not just `EcmaMetadata`?

Reply:
Oh, does it mean "Factory"? If so, this violates libraries style guide. We should call it `EcmaMetadataFactory`.

Reply:
Sorry. got lazy here. renamed to `XyzFactory`